docs-new/mkdocs.yml

```diff
@@ -0,0 +1,96 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+site_name: Apache Iceberg
+docs_dir: home
+
+theme:
+  name: material
+  language: en
+  logo: assets/images/iceberg-logo-icon.png
+  favicon: assets/images/favicon-96x96.png
+  features:
+    - navigation.tabs
+    - navigation.path
+    - navigation.top
+    - navigation.tracking
+    - toc.follow
+    - search.suggest
+    - search.highlight
+    - content.tabs.link
+    - content.code.copy
+    - content.code.annotate
+
+plugins:
+  - search
+  - macros:
+      include_yaml:
+        - variables.yml
+  - monorepo
+
+nav:
+  - Quickstart:
+    - Hive: hive-quickstart.md
+    - Spark: spark-quickstart.md
+  - Docs: 
+    - latest: '!include home/docs/latest/mkdocs.yml'
+    - 1.3.1: '!include home/docs/1.3.1/mkdocs.yml'
```

Comment:
We may be able to use the `*include` syntax here. There's an example in the mkdocs-monorepo-plugin README: `'*include ./components/*/mkdocs.yml'`.

Reply:
I tried this, but there's no way to control the ordering. I did already have to hack monorepo to get the dot syntax in the url to work, perhaps we add that as well?

Reply:
Sounds fine to me. I prefer to control ordering.

Reply:
As an aside, we're already going to need our own version of mkdocs monorepo until we can get the changes to accept '.' character in the url. Should we consider adding an ordering feature to this that would enable asc/desc for semver and ignoring non semver versions in the wildcard? That would make it one less thing we'd need to maintain and a valuable feature for a lot of folks trying to do the same thing we are.

Reply:
This wouldn't have to be now, but after we get this out add it to the roadmap.

Reply:
Let's open issues and discuss it with the mkdocs monorepo author. I don't think ordering is something we should be distracted by.